include/boost/geometry/formulas/vertex_longitude.hpp

```diff
@@ -147,10 +153,15 @@ class vertex_longitude_on_spheroid
         CT const C31 = (c1 - c1 * n) * eps + c2 * eps2;
         CT const C32 = c3 * eps2;
```

Comment:
Also here there are constants `C31` and `C32` which are conceptually somethign different, they are not representations of numbers `31` and `32`. So I think they should be named differently than other constants. So upper-case `C` is probably ok here since it makes the name different. The name could also be something more descriptive (like `sin2_sig2`).